GameFiles/GameInterface/_integration_t5.gsc

```diff
@@ -129,6 +129,11 @@ God()
     }
 }
 
+_GetXUID()
+{
```

Comment:
On the 3arc engine you do not need the call keyword to call a built-in by reference but it should be attempted anyway